packages/livechat/src/routes/Chat/component.js

```diff
@@ -238,7 +265,7 @@ class Chat extends Component {
 									)}
 								</ComposerActions>
 							}
-							limitTextLength={limitTextLength}
+							limitTextLength={!this.state.isRecording ? limitTextLength : null}
```

Comment:
```suggestion
							{...(!this.state.isRecording && {
								limitTextLength,
							})}
```

To avoid setting it as null, perhaps we can conditionally add the prop